frontend/next.config.js

```diff
@@ -47,13 +47,43 @@ const moduleExports = {
       ];
    },
    async redirects() {
-      return [
+      const redirects =[
          {
             source: '/Store/Guide/:guideid',
             destination: `${process.env.NEXT_PUBLIC_IFIXIT_ORIGIN}/Guide/_/:guideid`,
             permanent: true,
          },
       ];
+
+      // Do not hide /Parts and /Tools in dev
+      if (process.env.NODE_ENV == "development") {
+         return redirects;
+      }
+
+      return redirects.concat([
+         {
+            source: '/Parts',
+            has: [
+               {
+                 type: 'host',
+                 value: '^(?!www).*.*.*'
```

Comment:
This is the Magic™


See: regexr.com/6q3nv

Reply:
It's unclear to me from [the docs](https://nextjs.org/docs/api-reference/next.config.js/redirects#header-cookie-and-query-matching) what kind of regex `value` is (if it is a regex). Is the `.` a literal `.` here or a regex metacharacter? In either case, this doesn't seem right to me. If it's a literal `.`, then we're expecting 4 domain parts, but wouldn't we just have 3, like `de.ifixit.com`? If it's a metacharacter, then this would match basically any string that doesn't start with `www`, wouldn't it?

Here's an [updated regexr](https://regexr.com/6q9h7) with what I'm guessing is the intended regex, but I'm really not sure.